src/scheduler/round_robin.rs

```diff
@@ -16,14 +17,25 @@ impl RoundRobinScheduler {
     pub fn new() -> Self {
         Self {
             iterations: 0,
+            max_iterations: 1,
+            data_source: RandomDataSource::initialize(0),
+        }
+    }
+
+    /// Construct a new `RoundRobinScheduler` that will execute the test up to max_iteration times, scheduling its
+    /// tasks in a round-robin fashion.
+    pub fn new_multi_run(max_iterations : usize) -> Self {
```

Comment:
I think we can just drop these changes—we've been meaning to remove the RoundRobinScheduler for a while, and I don't think you're using it anywhere?